cmd/influxd/config.go

```diff
@@ -30,6 +30,12 @@ const (
 
 	// DefaultHTTPAPIPort represents the default port the HTTP API runs on.
 	DefaultHTTPAPIPort = 8086
+
+	// DefaultGraphitePort represents the default Graphite (Carbon) plaintext port.
+	DefaultGraphitePort = 2003
+
+	// DefaultGraphiteNameSeparator represents the default Graphite field separator.
+	DefaultGraphiteNameSeparator = "."
```

Comment:
Can you move these two `graphite`:

```
graphite.DefaultPort
graphite.DefaultNameSeparator
```

They're standard graphite pieces so it makes sense for them to live there. Especially if we make another CLI that needs to use the same defaults.